kratos/processes/apply_ray_casting_process.h

```diff
@@ -209,16 +179,14 @@ class KRATOS_API(KRATOS_CORE) ApplyRayCastingProcess : public Process
     ///@name Member Variables
     ///@{
 
+    Parameters mSettings;
     double mEpsilon = 1.0e-12;
     double mExtraRayOffset = 1.0e-8;
```

Comment:
I'd take the chance to make this user-definable by including them in the settings.

Reply:
This is being set here actually https://github.com/KratosMultiphysics/Kratos/blob/6aa01de1cfa6a5c56ddfff2ad16d422f1a3ffef0/kratos/processes/apply_ray_casting_process.cpp#L500

I will remove the default values as they are kind of confusing